prog/dftb+/lib_common/timerarray.F90

```diff
@@ -43,16 +46,43 @@ module timerarray
 contains
 
   !> Initializes a global timer
-  subroutine TTimerArray_init(this, timerItems)
+  subroutine TTimerArray_init(this, timerItems, maxLevel, header, unit)
 
     !> Instance
     type(TTimerArray), intent(out) :: this
 
     !> Names of the sub-timers to use
     type(TTimerItem), intent(in) :: timerItems(:)
 
+    !> Last level to be included (default: all levels are included)
```

Comment:
Last timer level to be included in printing / time evalualtion / ??? (default all ...